pkgs/servers/mail/listmonk/default.nix

```diff
@@ -12,8 +12,9 @@ buildGoModule rec {
   };
 
   patches = [
+    # Ensure that listmonk supports Go 1.20
     (fetchpatch {
-      url = "https://github.com/knadh/listmonk/pull/1479.patch";
+      url = "https://github.com/knadh/listmonk/commit/25513b81044803b104ada63c0be57a913960484e.patch";
       hash = "sha256-SYACM8r+NgeSWn9VJV4+wkm+6s/MhNGwn5zyc2tw7FU=";
     })
```

Comment:
I think adding the pr number or a link to the pr might be a good idea so it's easier for people to figure out that they can omit the patch when they update the package.

Reply:
It has been merged though, so the question is rather: is this commit part of an release, no?